elements/lisk-chain/src/chain.ts

```diff
@@ -124,14 +123,9 @@ export class Chain {
 		this._numberOfValidators = -1;
 		this.events = new EventEmitter();
 
-		this._accountSchema = objects.cloneDeep(baseAccountSchema);
-		this._defaultAccount = {};
-		for (const [name, schema] of Object.entries(accounts)) {
-			const { default: defaultValue, ...schemaWithoutDefault } = schema;
-			this._accountSchema.properties[name] = schemaWithoutDefault;
-			(this._accountSchema.required as string[]).push(name);
-			this._defaultAccount[name] = defaultValue;
-		}
+		const { defaultAccount, schema } = getAccountSchemaAndDefault(accounts);
```

Comment:
This argument here looks like accounts objects while the function accepts the account schema. This is related to earlier comment on other PR to rename `Chain` constructor arguments name `accounts` to `accountSchema`

Reply:
For the constructor param I added a point on https://github.com/LiskHQ/lisk-sdk/issues/5639